packages/onboarding/src/navigator/navigator-item-group/index.tsx

```diff
@@ -6,16 +6,18 @@ import './style.scss';
 
 type Props = {
 	children: React.ReactNode;
-	title: string;
+	title?: string;
```

Comment:
The file is copied from the pattern assembler. Make the title to be optional since we might only have a single group and it doesn't require a title.